classes/module/Module.php

```diff
@@ -1715,7 +1715,7 @@ final public static function isModuleTrusted($module_name)
             // Addons API and clear XML files to be regenerated next time
             static::deleteTrustedXmlCache();
 
-            return (int) Module::checkModuleFromAddonsApi($module_name);
+            return 0;
```

Comment:
Since the whole "trust feature" was based on an API cal to addons, which was removed here, it means the generated list will always contain only untrusted modules.

So this whole code could be replaced with `return false;` actually, so it raises the question: is the feature (and functions) still relevant? or should we remove all references to this feature?